src/cookies.rs

```diff
@@ -132,11 +79,10 @@ impl From<WebDriverCookie> for Cookie<'static> {
         }
 
         if let Some(same_site) = webdriver_cookie.same_site {
-            use cookie::SameSite as CookieSameSite;
-            cookie.set_same_site(match same_site {
-                SameSite::Strict => CookieSameSite::Strict,
-                SameSite::Lax => CookieSameSite::Lax,
-                SameSite::None => CookieSameSite::None,
+            cookie.set_same_site(match same_site.as_ref() {
```

Comment:
You could probably make this a little nicer with [`eq_ignore_ascii_case`](https://doc.rust-lang.org/std/primitive.str.html#method.eq_ignore_ascii_case)